README.md

```diff
@@ -0,0 +1,146 @@
+# Status endpoint
+Dit document is een richtlijn naar inrichting van de health check endpoint ingebouwd op alle REST API's. 
+Alle endpoints komen onder de resource `/status` terecht.
+
+## Health
+Deze endpoint beschrijft de werking van de service. Als deze niet werkende is zullen onderstaande niveaus worden gebruikt.
+
+#### Status niveau's
+Er zijn 3 verschillende niveaus van health status gedefinieerd.
+
+status|verklaring
+-------|-------
+ok|de service werkt 
+degraded|de service functioneert niet meer op 100% niet kritieke systemen zijn niet bereikbaar of de applicatie verwerkt trager requests
+outage|ernstige problemen die er toe leiden dat de service niet meer werkt.
+
+#### Basis response
+Default zouden we deze status aanbieden naar afnemers toe, deze endpoint zou mee publiek ontsloten zijn op de API Gateway. Omdat consumers alleen geïnteresseerd zijn in of de service werkt.
```

Comment:
Default zouden we volgende status aanbieden omdat consumers alleen willen weten of de service werkt. Dit endpoint wordt publiek mee ontsloten op de API Gateway.